src/forwarder.py

```diff
@@ -255,3 +256,6 @@ def set(self, result, *args):
 
     def debug_print(self):
         print(self._cache)
+
+
+message_forwarder = MessageForwarder(config.MAILBOX, config.HOME_SERVER, config.THREAD_ROLES)
```

Comment:
I don't see us wanting to create a MessageForwarder with any other configuration. Since we're already importing these config types, we could make the `__init__`  have no parameters and just set these member variables ourselves.

Reply:
The argument could even be made that MessageForwarder doesn't need to be an instantiated object, it could just be a collection of utility functions, saving us the trouble of keeping an object on hand, but that might be beyond the scope of this change.

Reply:
Yeah, same. I removed the member variables too - it just accesses the config fields directly where needed now.

Also yeah, I'd rather leave any refactor of the object to later.